resources/views/dashboard/controllers/events/index.blade.php

```diff
@@ -10,8 +10,15 @@
 <div class="container">
     @if(Auth::user()->isAbleTo('events'))
         <a href="/dashboard/admin/events/new" class="btn btn-primary">New Event</a>
-        <br><br>
     @endif
+
+
+    @if(Auth::user()->hasRole('events-team') || (Auth::user()->isAbleTo('events')))
+         @toggle('realops')
+            <a href="/dashboard/admin/realops" class="btn btn-primary">Realops</a>
```

Comment:
Will you please change this to "Realops Admin" just so the button's function is more descriptive? Thanks.